brats/transformations.py

```diff
@@ -73,3 +78,77 @@ def hist_match(source: np.ndarray, template: np.ndarray) -> np.ndarray:
                                     template_values)
         result[positive] = interp_t_values[bin_idx]
         return result
+
+
+class NiftiOrderTransformation:
+    """
+    Changes dimensions order from nifti (H,W,D,C) to torch convention (C,D,W,H).
+    It can be applied to both ``torch.Tensor`` or ``numpy.ndarray``.
```

Comment:
I think this line is redundant due to type hints and could only be a cause of false info in the future.

Reply:
It is already removed.